google-cloud-storage/src/test/java/com/google/cloud/storage/jqwik/StorageArbitraries.java

```diff
@@ -84,13 +85,39 @@ public static Arbitrary<String> randomString() {
     return Arbitraries.strings().all().ofMinLength(1).ofMaxLength(1024);
   }
 
+  public static Arbitrary<ProjectID> projectID() {
```

Comment:
Was it intentional to move this method up here? It's not showing if there is/what the change is from before

Reply:
I wanted to add it before bucketName because it's now part of the bucket proto. /shrug total nit. Other than that there should be no change.